endoreg_db/views/video/correction.py

```
@@ -0,0 +1,530 @@
+from rest_framework import status
+from django.shortcuts import get_object_or_404
+from django.conf import settings
+from pathlib import Path
+import json
+import logging
+
+from endoreg_db.models import VideoFile, VideoMetadata, VideoProcessingHistory, LabelVideoSegment
+from endoreg_db.serializers import VideoMetadataSerializer, VideoProcessingHistorySerializer
+from lx_anonymizer import FrameCleaner
+
+from endoreg_db.models import VideoFile
```

Ruff (0.14.1)

Redefinition of unused `VideoFile` from line 23

Remove definition: `VideoFile`

(F811)